plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestHiveGlueMetastore.java

```diff
@@ -310,6 +311,59 @@ public void testGetPartitions()
         }
     }
 
+    @Test
+    public void testGetPartitionsWithFilterUsingSpecialColumnName()
```

Comment:
Document what this test covers. For example, now there is no indication that `key` is not just a random name for partitioning column. Also there is no way to learn more.

Also, any reason this is Glue-specific?

Reply:
> Also, any reason this is Glue-specific?

The issue occurs for AWS Glue only AFAIK.

Reply:
I am aware, but the test could be generic.

Reply:
Please do give me a hint on how to make the test generic and still pinpoint the AWS Glue limitation.

Reply:
Can you have just same test code and move it to parent class?
the test data you can create via generic APIs (like create partitioned table with certain column names)

Reply:
I do use `io.trino.plugin.hive.AbstractTestHive#doCreateEmptyTable(io.trino.spi.connector.SchemaTableName, io.trino.plugin.hive.HiveStorageFormat, java.util.List<io.trino.spi.connector.ColumnMetadata>, java.util.List<java.lang.String>)`.

My initial intention was to use `createDummyPartitionedTable` method. However this method uses internally 

```
protected static final Predicate<String> PARTITION_COLUMN_FILTER = columnName -> columnName.equals("ds") || columnName.startsWith("part_");
```
reason why I decided to build manually the column metadata and the partition information.